RELEASE.md

```diff
@@ -0,0 +1,27 @@
+# Hdclient Release
+
+This document contains the steps to be followed to release
+the project hdclient.
+
+## Release Process
+
+To release hdclient:
+
+* Create a new branch: `git checkout -b feature/TICKET-123-release-x.y.z`
+
+* Name the tag for the repository.
+
+* Use the `npm version $tag` command with the same tag to update [package.json].
```

Comment:
We should be careful this creates the tag already. we should be careful then

Reply:
and this make the command `git tag` useless then because it already exsits

Reply:
What? Good catch, I didn't knew that, (I was used to `yarn version` which now I don't remember if it also tags the repo).

I'm gonna change it to, edit the file with the version, I like it more like that. (Even though tagging the branch and merging it is also a fair way of doing it).

Reply:
I've updated the document, now mentioning to edit the file. Thanks again for this, you saved me :')